### DISCUSSION TITLE
New Demo: SCADA - Piping and Instrumentation Diagram

### DISCUSSION THREAD

**Discussion by kumilingus** [clientIO/joint#2258] [News]
**Piping and Instrumentation Diagrams** (_P&IDs_) are integral components of **Supervisory Control and Data Acquisition** (_SCADA_) systems, providing a visual representation of the interconnected piping, equipment, and instrumentation in industrial processes. By incorporating real-time data and interactive features, _P&IDs_ enhance the monitoring, control, and maintenance capabilities of _SCADA_ systems, contributing to the efficient and safe operation of complex industrial facilities.

Today we've combined **shapes designed with gradients**, **animated views** of elements and links, **HTML form controls** attached to elements via highlighters, and **built-in charts** to display historical and current data. All of this is to create this visually vibrant and interactive [SCADA diagram](https://codepen.io/jointjs/full/oNQpZJV/094b9fea7c7248d1e29a12ffa5b37f55).

<img width="855" alt="image" src="https://github.com/clientIO/joint/assets/3967880/360fc744-54a2-4a5f-af74-6031eacb7836">

**Comment by wbzx** [comment]
I'm finding a electrical circuit diagram for animation

**Reply by kumilingus** [reply]
If you are looking for a new demo please use this [channel](https://github.com/clientIO/joint/discussions/1643) and provide us with as much information as you can.

Here's a [guide](https://github.com/clientIO/joint/wiki/Asking-Technical-Questions) on how to ask a good technical questions.

**Reply by alexandernst** [reply]
@kumilingus  Sorry for (kind of) hijacking this thread. Should I post https://github.com/clientIO/joint/discussions/2301#discussioncomment-6779903 in that channel? 🤔

**Reply by kumilingus** [reply]
@alexandernst it won't be necessary, but it's good of you to remind me 😄
